content/kapp-controller/docs/latest/package-authoring.md

```diff
@@ -89,9 +89,9 @@ reference to the greeter app image
 To start, lets create a directory with the above configuration files:
 
 ```bash
-$ mkdir -p package-contents/config/
-$ mv <path to config.yml> package-contents/config/config.yml
-$ mv <path to values.yml> package-contents/config/values.yml
+$ mkdir -p package-contents/
```

Comment:
I think maybe we should update the package bundles in the consumption workflow instead. Afaik using config dir is our suggested way. No rush on updating those assets, though, since the consumer workflow never exposes it.

Reply:
I changed the package bundle for version 1.0.0 to have this config/ dir and changed to ytt templating step to have a path for config/